docs/install/quickstart.md

```diff
@@ -16,7 +16,7 @@ RKE2 provides an installation script that is a convenient way to install it as a
 ```
 curl -sfL https://get.rke2.io | sh -
 ```
-This will install `rke2-server` and `rke2-agent` services onto your machine.
+This will install the `rke2-server` service and the `rke2` binary onto your machine.
```

Comment:
I had both services here because they do both get installed.

Do you think that’s just too much noise for a QuickStart?